server/src/routines/turn-actions.ts

```diff
@@ -375,3 +376,9 @@ export function* pickRequestSaga(
 
 	return
 }
+
+export function* delaySaga(game: GameModel, delayMs: number) {
+	if (game.viewers.length !== 0) {
+		yield* delay(delayMs)
+	}
+}
```

Comment:
This delays sending the game state until after the time elapses, it needs to begin the voice queue before waiting so it can finish by the time the next action is provided

Reply:
I added the `sendGame` function before the `delaySaga`. This should be okay, as during tests the send function will do nothing because there is no viewers.